types/KBaseClassifier.GenomeCategorizer/display.yaml

```diff
@@ -0,0 +1,14 @@
+#
+# Define basic display information
+#
+name     : GenomeCategorizer
+subtitle : Genome Categorizer object
+tooltip  : Genome Categorizer object
+
+icon : none
+
+description : |
+    <p>This type contains a ML model .</p>
```

Comment:
A what? Machine Learning? Maximum Likelihood? Monocytogenes Listeria? (Ok, I made that up)